src/screens/ChannelListScreen/index.tsx

```diff
@@ -44,6 +46,7 @@ const ChannelListScreen = ({route}) => {
 
   React.useEffect(() => {
     if (isFocused) {
+      getAllSignedChannels();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getAllSignedChannels` function is being called without any arguments, but according to the summary of changes, it now accepts an optional `timeStamp` parameter. If this parameter has a significant role in the function's behavior, consider passing it when calling the function. If the timestamp is not necessary in this context, then the code is fine as it is.
```diff
-      getAllSignedChannels();
+      getAllSignedChannels(timeStamp);
```
Please replace `timeStamp` with the actual timestamp value or variable you want to use.


<!-- This is an auto-generated comment by OSS CodeRabbit -->